packages/interbit-ui-tools/src/middleware/browserContext/index.js

```diff
@@ -0,0 +1,11 @@
+const { getInterbit, waitForInterbit, isInterbitLoaded } = require('./context')
+const getHtmlConfig = require('./staticHtml')
+
+const getConfig = async () => getHtmlConfig(document)
+
+module.exports = {
+  getConfig,
+  getInterbit,
+  waitForInterbit,
+  isInterbitLoaded
+}
```

Comment:
This is the interface for the `runtimeContext`. Test mocks will need to implement this.